Packs/CoreAlertFields/IncidentFields/incidentfield-Email_Subject.json

```diff
@@ -0,0 +1,38 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "emailsubject",
+    "closeForm": false,
+    "content": true,
+    "description": "The email subject value of a firewall alerts triggered on a the content of a malicious email.",
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_emailsubject",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Email Subject",
+    "neverSetAsRequired": false,
+    "openEnded": true,
+    "ownerOnly": false,
+    "propagationLabels": [
+        "all"
+    ],
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+    "x2_fields": "fw_email_subject",
+    "Aliases": [
+        {
+          "cliName": "emailsubject",
```

Comment:
I think this can be a problem - 
the alias cliname is the same as the current field (emailsubject), i guess you mean the xsoar one but the names are identical
@yaakovi @shaniacht1 will this work?

Reply:
@amit-dror1993 can you please confirm if this is ok?